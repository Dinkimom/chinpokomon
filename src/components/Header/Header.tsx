import React from 'react';
import './Header.css';
import pokeball from './pokeball.png';

export const Header: React.FC = () => (
  <div className="header">
    <h1>
      Pokedesk <span>mini</span>
    </h1>

    <img src={pokeball} alt="pokeball" width="30px" height="30px" />
  </div>
);
