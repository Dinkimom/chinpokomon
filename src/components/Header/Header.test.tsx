import React from 'react';
import { shallow } from 'enzyme';

import { Header } from './Header';

describe('<Header /> : initial', () => {
  it('should render correctly', () => {
    const component = shallow(<Header />);

    expect(component).toMatchSnapshot();
  });
});
