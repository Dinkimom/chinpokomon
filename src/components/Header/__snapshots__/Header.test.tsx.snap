// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> : initial should render correctly 1`] = `ShallowWrapper {}`;
